### PR TITLE
Implemented new service-interface-base interfaces

### PR DESCRIPTION
**Pull request by rhyswilliamsza**
> We want to be able to implement logic that can do payment orchestration for both Airtime and PPU messages polymorphically.

We have added `HasAmounts` & `HasPaymentMethods` interfaces' in service-interface-base in order to solve the above problem statement. This PR simply pulls these changes into the prepaidutility-service-interface.

Imports have changed in the modified files to conform to Electrums Eclipse Code Formatter profile. Please let me know if I should change it back. Note, too, the minor version bump due to the reliance on these new interfaces.

### CODE REVIEW

File: src/main/java/io/electrum/prepaidutility/model/MeterLookupResponse.java [hunk 1]
```diff
@@ -48,8 +43,8 @@ public void setMeter(Meter meter) {
       this.meter = meter;
    }
 
-   public MeterLookupResponse customer(Customer customer) {
-      this.customer = customer;
+   public MeterLookupResponse meter(Meter meter) {
+      this.meter = meter;
       return this;
```
Comment (MNS003) [inline src/main/java/io/electrum/prepaidutility/model/MeterLookupResponse.java:48]: Did you re-order code?

File: src/main/java/io/electrum/prepaidutility/model/MeterLookupResponse.java [hunk 2]
```diff
@@ -130,8 +125,28 @@ public void setMaxAmount(LedgerAmount maxAmount) {
...
+   /**
+    * Maximum purchase amount that can be requested by the customer.
+    *
+    * @return maxAmount
+    **/
+   @ApiModelProperty(value = "Maximum purchase amount that can be requested by the customer.")
+   @Valid
+   public LedgerAmount getArrearsAmount() {
+      return arrearsAmount;
+   }
```
Comment (MNS003) [inline src/main/java/io/electrum/prepaidutility/model/MeterLookupResponse.java:142]: Comment does not seem to belong to code.

File: src/main/java/io/electrum/prepaidutility/model/MeterLookupResponse.java [hunk 3]
```diff
@@ -151,25 +166,29 @@ public void setBsstDue(Boolean bsstDue) {
...
+      sb.append(", arrearsAmount=").append(arrearsAmount);
+      sb.append(", bsstDue=").append(bsstDue);
+      sb.append(", id='").append(id).append('\'');
+      sb.append(", time=").append(time);
+      sb.append(", originator=").append(originator);
+      sb.append(", client=").append(client);
+      sb.append(", settlementEntity=").append(settlementEntity);
+      sb.append(", receiver=").append(receiver);
+      sb.append(", thirdPartyIdentifiers=").append(thirdPartyIdentifiers);
+      sb.append('}');
```
Comment (MNS003) [inline src/main/java/io/electrum/prepaidutility/model/MeterLookupResponse.java:191]: Why the changes (":" vs "=", order of attributes, no more line-breaks)?

File: src/main/java/io/electrum/prepaidutility/model/PurchaseResponse.java [hunk 4]
```diff
@@ -72,6 +75,17 @@ public void setTaxTotal(LedgerAmount taxTotal) {
...
+   @ApiModelProperty(required = false, value = "An optional amounts field for any additionalAmounts which may need to be added to the response payload.")
+   @Override
+   public Amounts getAmounts() {
+      return additionalAmounts;
+   }
+
+   @Override
+   public void setAmounts(Amounts amounts) {
+      this.additionalAmounts = amounts;
+   }
```
Comment (MNS003) [inline src/main/java/io/electrum/prepaidutility/model/PurchaseResponse.java:87]:
```suggestion
   @Override
   public void setAmounts(Amounts amounts) {
      this.additionalAmounts = amounts;
   }
   
   public void additionalAmounts(Amounts additionalAmounts) {
      this.additionalAmounts = additionalAmounts;
   }
```

File: src/main/java/io/electrum/prepaidutility/model/MeterLookupResponse.java (hunk 1 above)
Comment (rhyswilliamsza) [reply src/main/java/io/electrum/prepaidutility/model/MeterLookupResponse.java:48]: I did, however it was necessary for grouping unfortunately. Doesn't change any core code :)

File: src/main/java/io/electrum/prepaidutility/model/MeterLookupResponse.java [hunk 5]
```diff
@@ -130,8 +125,28 @@ public void setMaxAmount(LedgerAmount maxAmount) {
...
+      this.maxAmount = maxAmount;
+      return this;
+   }
+
+   /**
+    * Maximum purchase amount that can be requested by the customer.
+    *
+    * @return maxAmount
+    **/
+   @ApiModelProperty(value = "Maximum purchase amount that can be requested by the customer.")
```
Comment (rhyswilliamsza) [inline src/main/java/io/electrum/prepaidutility/model/MeterLookupResponse.java:138]:
```suggestion
   @ApiModelProperty(value = "Returned arrears amount from provider. Encapsulates the total debt owed by the customer.")
```
